torch/tapir_model.py

```diff
@@ -193,18 +206,12 @@ def forward(
     )
 
     p = self.num_pips_iter
-    out = dict(
-        occlusion=torch.mean(
-            torch.stack(trajectories['occlusion'][p::p]), dim=0
-        ),
-        tracks=torch.mean(torch.stack(trajectories['tracks'][p::p]), dim=0),
-        expected_dist=torch.mean(
-            torch.stack(trajectories['expected_dist'][p::p]), dim=0
-        ),
-        unrefined_occlusion=trajectories['occlusion'][:-1],
-        unrefined_tracks=trajectories['tracks'][:-1],
-        unrefined_expected_dist=trajectories['expected_dist'][:-1],
-    )
+    
+    # change the code below to make it torch.jit.trace/torch.jit.script friendly
+    out = (torch.mean(torch.stack(trajectories.occlusion[p::p]), dim=0),
+                 torch.mean(torch.stack(trajectories.tracks[p::p]), dim=0),
+                 torch.mean(torch.stack(trajectories.expected_dist[p::p]), dim=0)
+          )
```

Comment:
If there is a use case to return `unrefined_occlusion`, `unrefined_tracks` and `unrefined_expected_dist` then the code should be modified as 
```
    out = (torch.mean(torch.stack(trajectories.occlusion[p::p]), dim=0),       # occlusion
           torch.mean(torch.stack(trajectories.tracks[p::p]), dim=0),          # tracks
           torch.mean(torch.stack(trajectories.expected_dist[p::p]), dim=0),   # expected_dist
           trajectories.occlusion[:-1],                                        # unrefined_occlusion
           trajectories.tracks[:-1],                                           # unrefined_tracks
           trajectories.expected_dist[:-1]                                     # unrefined_expected_dist        
          )
```
having `-> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:` replaced with `-> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, list[torch.Tensor], list[torch.Tensor], list[torch.Tensor]]:` in the `forward` annotation.